shared/stats/src/index.mjs

```diff
@@ -44,12 +56,31 @@ export async function getStats (modulePath) {
   // Modules bundled
   const modules = stats?.[base] ?? {}
 
+  const moduleCount = Object.keys(modules).length
+
   // Modules total size
   const total = Object.values(modules)
     .map(({ rendered }) => rendered)
     .reduce((total, rendered) => total + rendered, 0)
 
-  return { total, modules }
+  return { total, modules, moduleCount }
+}
+
+/**
+ * Returns file sizes of key files
+ *
+ * @returns {Promise<{[key: string]: import('fs').Stats}>} - File names and size
+ */
+export async function getFileSizes () {
+  /** @type { { [key: string]: import('fs').Stats } } */
+  const result = {}
+
+  for (const filename of filesForAnalysis) {
+    const stats = await stat(join(paths.root, filename))
```

Comment:
Might need a tweak to account for different versions.

Reply:
Yeah it's a bit of a worry that we're using `pkg.version` and `paths.root`

For example, we could have [GOV.UK Frontend v4.7.0 installed in the review app + stats packages](https://github.com/alphagov/govuk-frontend/blob/main/shared/stats/README.md#stats-for-previous-version):

```shell
npm install govuk-frontend@4.7.0 --workspace packages/govuk-frontend-review
npm install govuk-frontend@4.7.0 --workspace shared/stats
```

The include `<script type="module">` wouldn't work but the names, files, data would all be correct

We'd see stats coming from `shared/stats/node_modules/govuk-frontend` not `packages/govuk-frontend`. Makes it difficult to maintain `filesForAnalysis` doesn't it 😣 

Could we just stick to component module file sizes for now?

Then we only need to loop `componentNamesWithJavaScript` regardless of version:

https://github.com/alphagov/govuk-frontend/blob/5804103ad5d417222f3b114c8c2db4e694ba2801/packages/govuk-frontend-review/src/views/index.njk#L116-L117

Reply:
I've made sure all helpers work with any GOV.UK Frontend version in:

* https://github.com/alphagov/govuk-frontend/pull/4041

So it might be nice to move the filesize helpers to `govuk-frontend-lib/files` too

Reply:
`getFileSizes()` now takes a glob as input, which I'm passing from the `comments` script. It should be sizing everything in `dist` and the files directly in `packages/govuk-frontend/dist/govuk`. In both cases, ignoring `.map` files.